src/demos/error/error-demo.component.html

```diff
@@ -112,9 +112,13 @@ <h3 class="sky-subsection-heading">
   Display an error message in a modal
 </h3>
 
+<p>
```

Comment:
I am more confused about the error modal now after looking most closely at the demo and properties. The component seems to not be implemented in a way that follows UX guidelines.

Reply:
@Blackbaud-ToddRoberts What is wrong with the implementation? And does it impact this PR? If it doesn't impact this PR, then can you file a separate issue about the implementation?

Reply:
The guideline here to not use images in modals doesn't follow UX guidelines laid out on the Error page and Error Handling guidelines, but it looks like the error modal component does not allow an image?

Reply:
Todd, I think you're right about the error modal not allowing an image. @Blackbaud-BobbyEarl, would you mind confirming? (I tagged Bobby because I asked him for help differentiating between regular modals and the error modal.)

Based on Bobby's explanation to me and the questions in this thread, sounds like we need guidelines about when to use the error modal specifically vs. a modal following the error handling guidelines.

Reply:
The error modal only allows for a title, description, and controlling the text used in the "close button."

I was not involved with the implementation of this nor do I really recall why it exists, but it is not mentioned in current [error handling](https://developer.blackbaud.com/skyux/design/guidelines/error-handling) docs at all.  I think determining where is it used and should we keep it around is most important.

Assuming the answer to that question is yes, we should then focus on updating the documentation to reflect when to use it.

Reply:
Sounds like this is something for the team to discuss during flextime. @Blackbaud-AlexKingman, can you lead that conversation? Or do we need more info from Bobby?

Reply:
After discussing this with the team, the decision was made to pull the guidance and leave the content as is. I'm going to resolve the conversation.